packages.yml

```
@@ -1,5 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=1.1.1", "<2.0.0"]
-  - package: metaplane/dbt_expectations
-    version: 0.10.8 # Check
\ No newline at end of file
+    version: [">=1.1.1", "<2.0.0"]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)